packages/api-headless-cms-ddb/src/operations/model/index.ts

```diff
@@ -19,6 +19,11 @@ interface PartitionKeysParams {
 }
 const createPartitionKey = (params: PartitionKeysParams): string => {
     const { tenant, locale } = params;
+    if (!tenant) {
```

Comment:
Saw that you ended your error messages with a "." above, so let's end it here too. :)